### PR TITLE
Update plot.mob_out and plot.mob_stats and add overlap_effects

### PR DESCRIPTION
**Pull request by dmcglinn**
These commits update the specification of colors in the functions `plot.mob_out` and `plot.mob_stats`. Also a new graphical summary function was added `overlap_effects` which replaces the older function `stack_effects`. The new function allows users to examine the raw effects as well as the stacked effects of the absolute value of the effects.

### CODE REVIEW

**Comment by dmcglinn** [comment]
Thanks @FelixMay. Let's not merge just yet I have one minor commit to add to this PR.

File: R/mobr.R [hunk 1]
```diff
@@ -1422,25 +1423,37 @@ plot.mob_out = function(mob_out, trt_group, ref_group, same_scale=FALSE,
...
+#' 
+#' All three treatment effect sizes due to the SAD, N, or aggregation are
+#' graphed on a single plot. The treatment effect is defined as the treatment difference
+#' in richness due to a particular componnet of community structure. The overlap
+#' of the three effects is accomplished by rescaling numbers of individuals 
+#' to the number of plots. The effect sizes may be plotted simply in their raw 
+#' form or as stacked area plots. 
+#' @param display the type of graphic to display options include: "raw" or
+#'  "stacked" for plots of the raw effect sizes or a stacked area plot of the
+#'  absolute value of the effect sizes.
```
Comment (rueuntal) [inline R/mobr.R]: Maybe add default is "raw".

File: R/mobr.R [hunk 2]
```diff
@@ -1478,10 +1496,15 @@ stack_effects = function(mob_out, trt_group, prop=FALSE,
         effort = min(agg$effort):round(N_indiv / N_plots)
         virt_effort = effort * plot_dens
     }
-    SAD_interp = pracma::pchip(SAD$effort, SAD$effect, virt_effort)
-    N_interp = pracma::pchip(N$effort, N$effect, virt_effort)
-    SAD = data.frame(type='SAD', effort, effect=SAD_interp)
-    N = data.frame(type='N', effort=effort, effect=N_interp)
+    if (stretch) {
```
Comment (rueuntal) [inline R/mobr.R]: Is there a reason not to combine the two `if (stretch) else` sections?

**Comment by rueuntal** [comment]
Hey @dmcglinn - great work! I have two very minor comments but everything else look good.

**Comment by dmcglinn** [comment]
I fixed your minor comments @rueuntal thanks for those! Merging now.

**Pull request merged**